code/datums/craft/recipes/lodge.dm

```diff
@@ -52,6 +52,61 @@
 		list(/obj/item/reagent_containers/food/snacks/grown, 3, "time" = 1) //Takes a second
 	)
 
+/datum/craft_recipe/lodge/throwing_knife/bone_needle
+	name = "throwing bone needles"
+	result = /obj/item/stack/thrown/throwing_knife/bone_needle
+	steps = list(
+		list(CRAFT_MATERIAL, 3, MATERIAL_BONE, "time" = 1), //Takes a second
+		list(QUALITY_SAWING, 20, "time" = 5),
+		list(QUALITY_WELDING, 30, "time" = 60),
```

Comment:
Welding the...bones? Maybe change this to something else since the recipe doesn't include metal.